### ISSUE TITLE
ModuleNameMapper duplicate path

### ISSUE THREAD

**Issue by IncredibleMrTim** [risenforces/cra-alias#4]
Hey Guys

There seems to be an issue where the `moduleNameMapper` path is replicated

```
    Please check your configuration for these entries:
    {
      "moduleNameMapper": {
        "/^@components(.*)$/": "/Users/xxx/Documents/Projects/react/xxx/xxx-ui/src/Users/xxx/Documents/Projects/react/xxx/xxx-ui/src/components$1"
      },
      "resolver": /Users/xxx/Documents/Projects/react/xxx/xxx-ui/node_modules/react-scripts/node_modules/jest-pnp-resolver/index.js
    }
```
this is preventing me from running any of my unit tests

jsconfig.json
```
{
  "compilerOptions": {
    "baseUrl": "src",
    "paths": {
      "@features": [
        "./features"
      ],
      "@components": [
        "components"
      ],
      "@constants": [
        "./constants"
      ],
      "@context": [
        "./context"
      ],
      "@reducers": [
        "./reducers"
      ],
      "@services": [
        "./services"
      ],
      "@hooks": [
        "./hooks"
      ]
    }
  }
}
```

**Comment by risenxxx** [comment]
Hi @IncredibleMrTim, thanks for the issue. I will check it soon. A question: does it work with `./` before components? But, in any case, I wonder why part of the path is duplicated, so I will look for the reason.

**Comment by IncredibleMrTim** [comment]
Hey @risenforces yes it does work with `./`.  I removed this to test if that was the issue.  Thanks for looking into this for me.

**Comment by risenxxx** [comment]
Fixed in 4.13.16 (https://github.com/risenforces/cra-alias/commit/796e63cf571e52d8dc2222bf9e553d9af6015aed + https://github.com/risenforces/cra-alias/commit/c4c3f9e463f54a6c462c926fe93da107f20ccac0)
@IncredibleMrTim
